README.md

```diff
@@ -61,14 +62,23 @@ Update any relevant keys in your new .env file. For example, if you will be work
 
 Please note: Any time you change the values in this .env file, you will need to stop and restart your Docker container in order for the new values to take effect.
 
-#### Install Docker Compose on Mac
+#### Install Docker Desktop
```

Comment:
thanks for adding. why do you think "Install Docker Compose on Mac" should be removed?